packages/output/src/browser/output-contribution.ts

```diff
@@ -61,6 +61,11 @@ export namespace OutputCommands {
     export const CLEAR__WIDGET: Command = {
         id: 'output:widget:clear',
         category: OUTPUT_CATEGORY,
+    };
+
+    export const CLEAR__WIDGET_TOOLBAR: Command = {
+        id: 'output:widget:toolbar:clear',
```

Comment:
Why cannot we improve `output:widget:clear` command instead?

Reply:
I tried, but did not manage to get an expression that works for the button and the contextmenu at the same time. So I checked for other cases and in the debug-console the Button 'Clear Console' has a different command than the contextmenu option. So I went with that. How whould you implement it?

Reply:
I am with @akosyakov not to introduce too many unnecessary commands, on the other hand, @salkin-naj is right; if we add a new command we do not have to overcomplicate the handler rule.

Reply:
ok

Reply:
Yes, it is not exactly the same. 
Before my fix, both would use the same command, but because of the handler rule the command was not visible in the context menu. As I said I tried to add a condition, that would fit the context menu  as well. But when the context menu is opened and the handler checks, if the command should be visible, I did not manage to access the widget. All i managed to get was the x/y coordinates where the contextmenu was opened. Maybe you could check, if the coordinates are over the output view, but this did not seem to be a great solution to me. 
So I thought this was a better solution. If there is a way to check, if the context menu was opened inside the output view, that I missed, please let me know.